GiniSwitchSDK/GiniSwitchSDK/Classes/ExtractionsManager.swift

```diff
@@ -287,7 +288,12 @@ class ExtractionsManager {
         status?.pages.forEach({ (page) in
             if let pageRef = page.href,
                 let scannedPage = scannedPages.page(for: pageRef) {
-                if scannedPage.status != page.pageStatus {
+                // ignore statue changes for pages scheduled for deletion and
+                // replacement. They might be marked as analysed, but since they
+                // will be replaced soon, the status shouldn't be displayed
+                if scannedPage.status != page.pageStatus &&
+                    scannedPage.status != .replaced &&
+                    scannedPage.status != .deleted {
```

Comment:
would be nice it this is in a function `if shouldDisplayStatus()…`
also typo in the comment `statue`